### DISCUSSION TITLE
When zoom out in ghostty while running `+list-themes` causes a segmentation error.

### DISCUSSION THREAD

**Discussion by m4saurabh** [ghostty-org/ghostty#3692] [Feature Requests, Ideas] [answered]
I was just messing around with ghostty and I noticed this.
There is no reason zoom out that much but it should not cause a segmentation error.

https://github.com/user-attachments/assets/672ba0c8-6243-4269-8f2d-6bdf79e9dd05

**Comment by 00-kat** [comment]
I can also reproduce this.

(I hope you don't mind if I ping you, @jcollie @rockorager)

**Comment by rhodes-b** [comment, marked as answer]
I can reproduce this on 1.1.3 but not on tip this is solved?
